packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -993,54 +993,23 @@ export const isTtlModified = (
  * @returns time to live specification object
  */
 const getTtlStatus = async (tableName: string): Promise<DescribeTimeToLiveCommandOutput> => {
-  const initialDelay = Math.floor(Math.random() * 10 * 1000); // between 0 to 10s
+  const initialDelay = Math.floor(Math.random() * 5 * 1000); // between 0 to 5s
```

Comment:
Thank you for testing this. Let's add a brief doc comment explaining why we're using an initial delay rather than just retrying on failure.